doc/source/install.rst

```diff
@@ -47,13 +47,43 @@ Execute the following command from the shell::
 
 Windows
 ```````
+Before installing the development version, uninstall the standard version of
```

Comment:
I think that with virtual environments there are plenty of different combinations to consider and describe. How about adding something like this at the beginning instead: "We will consider the case where the used has a default virtual Python environment configured and would like to install the dev version inside it. For more complex cases [send the reader to some resources on venv, pyenv, etc]"